pkg/alertmanager/validation/v1alpha1/validation.go

```diff
@@ -208,7 +207,7 @@ func validateEmailConfig(configs []monitoringv1alpha1.EmailConfig) error {
 		if config.Smarthost != "" {
 			_, _, err := net.SplitHostPort(config.Smarthost)
 			if err != nil {
-				return errors.Wrapf(err, "invalid field 'smarthost': %s", config.Smarthost)
+				return fmt.Errorf("invalid 'smarthost' %s: %w", config.Smarthost, err)
```

Comment:
This is the only logline that I made a significant change. It already had `:`, so I re-arranged it.

(same in v1beta1)

Reply:
this looks good to me.